src/stylesheets/components/_search.scss

```diff
@@ -60,7 +60,7 @@
   [type="submit"],
   .usa-search__submit {
     @include u-padding-x(1.5);
-    width: units($theme-button-small-width);
+    min-width: units($theme-button-small-width);
```

Comment:
Updates to min-width here to allow button to accommodate inline image. Without it, a fixed-width button could prevent the image from displaying at full size